### PR TITLE
Completed JS Forms - AT

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-02-03 at 12 17 19 PM
Screen Shot 2021-02-03 at 12 01 52 PM

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+var $inputArray = document.querySelectorAll('input');
+var $inputName = $inputArray[0];
+var $inputEmail = $inputArray[1];
+var $textcontent = document.querySelector('textarea');
+
+$inputName.addEventListener('focus', handleFocus);
+$inputEmail.addEventListener('blur', handleBlur);
+$textcontent.addEventListener('input', handleInput);
```
Comment (codingcodymiller) [inline javascript-forms/form-events.js:22]: Screen Shot 2021-02-03 at 11 21 06 AM

You are missing some event handlers here. Each form control should have an event listener for each of these events.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! Please see my note:

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-03 at 12 30 38 PM

**Review by codingcodymiller — APPROVED** [review]
Nicely done!

**Pull request merged**